File: yahtzee_v20_yahtzee/Dice.cpp
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */

#include "Dice.h"

#include <cstdlib>
#include <ctime>

Dice::Dice() {
    value = 0;
    srand(static_cast<unsigned int>(time(0))); // random numeber generator
}

void Dice::roll() {
    value = rand() % 6 + 1;
}

int Dice::getValue() const {
    return value;
}

File: danielle_yahtzee_htmlCode/yahtzee_code_html/yahtzee_v4/public_html/Dice.js

function Dice() {   
    //Public properties of the class
    var value = 0;
    alert("Hit Dice(). value = " + value);
}

 //Dice.prototype.roll=function(){
 function roll(){
    //alert("Hit roll()"); 
    value = (1 + (Math.floor(Math.random() * 6)));
    alert("Hit roll(). dice = " + value);
 };


Dice.prototype.getValue=function() {
//function getValue(){
    return value;
};